dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -10,6 +10,7 @@
 - hide_source ||= false
 - use_phaser ||= false
 - use_makerlab ||= false
+- app_options[:locale] = js_locale
```

Comment:
according to a quick diff of `appOptions` on [test](https://test-studio.code.org/s/frozen/stage/1/puzzle/2) vs [prod](https://studio.code.org/s/frozen/stage/1/puzzle/2), we are also potentially missing `disableSocialShare`.